include/z64actor.h

```diff
@@ -231,55 +254,90 @@ typedef struct {
+    /* 0x06BE */ s16        unk_6BE;
+    /* 0x06C0 */ s16        unk_6C0;
+    /* 0x06C2 */ s16        unk_6C2;
+    /* 0x06C4 */ f32        unk_6C4;
     /* 0x06C8 */ SkelAnime  skelAnime2;
     /* 0x070C */ char       unk_70C[0x128];
     /* 0x0834 */ s16        unk_834;
     /* 0x0836 */ char       unk_836[0x002];
-    /* 0x0838 */ f32        unk_838;
-    /* 0x083C */ s16        unk_83C;
-    /* 0x083E */ char       unk_83E[0x004];
+    /* 0x0838 */ f32        linearVelocity;
+    /* 0x083C */ s16        currentYaw;
+    /* 0x083E */ s16        targetYaw;
+    /* 0x0840 */ u16        unk_840;
     /* 0x0842 */ s8         swordAnimation;
     /* 0x0843 */ s8         swordState;
     /* 0x0844 */ u8         unk_844;
     /* 0x0845 */ u8         unk_845;
     /* 0x0846 */ u8         unk_846;
     /* 0x0847 */ char       unk_847[0x004];
-    /* 0x084B */ s8         unk_84B[UNK_SIZE];
-    /* 0x084C */ char       unk_84C[0x003];
+    /* 0x084B */ s8         unk_84B[4];
     /* 0x084F */ s8         unk_84F;
     /* 0x0850 */ s16        unk_850;
-    /* 0x0852 */ char       unk_852[0x04E];
+    /* 0x0852 */ char       unk_852[0x00A];
+    /* 0x085C */ f32        stickLength;
+    /* 0x0860 */ s16        stickFlameTimer;
+    /* 0x0862 */ s8         overheadItemId;
+    /* 0x0863 */ char       unk_863[0x021];
+    /* 0x0884 */ f32        ledgeDistance; // The distance from link to a grabbable ledge, only updates if pushing against a wall with a grabbable ledge above, if the ledge is too high to grab the value is 399.96f
+    /* 0x0888 */ f32        wallDistance; // Only updates if pushing against a wall with a grabbable ledge above
+    /* 0x088C */ char       unk_88C[0x008];
+    /* 0x0894 */ s16        dropY; // Truncated copy of y position that does not update while falling
+    /* 0x0896 */ s16        fallY; // The truncated y distance link has moved in that frame, positive is down, negative is up
+    /* 0x0898 */ char       unk_898[0x008];
     /* 0x08A0 */ u8         unk_8A0;
     /* 0x08A1 */ u8         unk_8A1;
     /* 0x08A2 */ u16        unk_8A2;
     /* 0x08A4 */ f32        unk_8A4;
     /* 0x08A8 */ f32        unk_8A8;
-    /* 0x08AC */ char       unk_8AC[0x174];
+    /* 0x08AC */ f32        fanWindSpeed;
+    /* 0x08B0 */ s16        fanWindDirection;
+    /* 0x08B2 */ char       unk_8B2[0x002];
+    /* 0x08B4 */ Struct_80090480_arg2 swordDimensions; // Trail active, tip, base?
+    /* 0x08D0 */ Struct_80090480_arg2 unk_8D0;
+    /* 0x08EC */ Struct_80090480_arg2 unk_8EC;
+    /* 0x0908 */ char       unk_908[0x118];
     /* 0x0A20 */ MtxF       mf_A20;
-    /* 0x0A60 */ char       unk_A60[0x18];
-    /* 0x0A78 */ s8         unk_A78;
+    /* 0x0A60 */ char       unk_A60[0x08];
+    /* 0x0A68 */ s8         unk_A68;
+    /* 0x0A69 */ char       unk_A6A[0x0F];
+    /* 0x0A78 */ s8         invincible; // Take no damage if this value is nonzero, positive induces red flashing, negative is "invisible" invincibility
```

Comment:
i missed this in the last review, but I would call this invincibility or even invincibilityTimer. it counts up or down to 0 depending on if its positive or negative, and link is invincible during those frames when the timer is active.
Other than that this is good to go for me